user/languages/python.md

```diff
@@ -192,6 +192,8 @@ script: python3 my_app.py || python my_app.py
 
 ### pip
 
+> As per the [PyPa](https://pip.pypa.io/en/latest/user_guide/#changes-to-the-pip-dependency-resolver-in-20-2-2020) announcement, we will be shipping out `pip` version 20.3 in our build images. We advise you test the effects of this version of `pip` on your builds as advised in the linked page.
```

Comment:
An English suggestion to avoid repetition (changes in bold): 

We **recommend** you test the effects of this **'pip' version on** your builds as advised on the linked page.

Reply:
Thanks for the suggestion. I've opted to expand this portion a little bit to give more context.